recipe/meta.yaml

```diff
@@ -8,43 +8,80 @@ package:
 
 source:
   url: https://github.com/ec-jrc/pyPoseidon/archive/{{ version }}.tar.gz
-  sha256: db16f8215bbd534b2c0bce5617894aebdecd76b88016618ba8ce220dca75005e
+  sha256: 89a975a60af560ffd3c645056cca530961f6fefd5541cd042052d4159abec093
 
 build:
   noarch: python
   number: {{ build }}
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
-requirements:
-  build:
-  host:
-    - pip
-    - python >=3.6
-    - setuptools
-    - poetry
-    - poetry-core
-  run:
-    - python >=3.6
-    - mayavi
-    - pyresample
-    - pygeos >=0.7.1
-    - geoviews
-    - rasterio
-    - jigsaw =0.9.12
-    - hvplot
-    - descartes
-    - cfgrib <0.9.9
-    - moviepy
-    - colorlog
-    - f90nml
-    - zarr
+outputs:
+  - name: pyposeidon-base
+    script: build_base.sh
+    build:
+      noarch: python
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+      host:
+        - pip
+        - python
+      run:
+        - python
+        - pyresample
+        - pygeos >=0.10
+        - rasterio
+        - rioxarray
+        - jigsaw
+        - cfgrib
+        - colorlog
+        - f90nml
+        - zarr
+        - gmsh >=4.8.4
+        - python-gmsh
+        - tqdm
+        - xarray
+        - pandas
+        - geopandas >=0.10.1
+        - numpy
+        - psutil
+        - requests
+        - scipy
+        - netCDF4
+        - dask
+        - shapely =1.7.1
+        - pydap >=3.3.0
+
+    test:
+      imports:
+        - pyposeidon
+
+  - name: pyposeidon
+    requirements:
+      host:
+        - pip
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('pyposeidon-base', exact=True) }}
```

Comment:
I think `exact` was problematic in multiple outputs. Maybe use something like `max_pin='x.x.x'`?

Reply:
I have tried it locally and the result is the same. When I was testing the build on my own channel I didn't have tests. It turns out that when I activate the test import it fails to build.

Without it, it builds and resolves afterwards correctly. I can't figure out why. Any tips are welcome but for now I will just commend out the test.